ui/tests/acceptance/dashboard-test.js

```diff
@@ -381,7 +382,7 @@ module('Acceptance | landing page dashboard', function (hooks) {
       await runCmd([`write sys/mounts/kv1 type=kv`]);
       await settled();
       await visit('/vault/dashboard');
-      await click('[data-test-component="search-select"] .ember-basic-dropdown-trigger');
+      await clickTrigger('#type-to-select-a-mount');
```

Comment:
A bit trivial but any reason not to put a test selector with a unique identifier on the `SearchSelect` component? If the `QuickActionsCard` component is used multiple times in the same view then that id will be duplicated.

Reply:
ah, good call out. Let me add that as a test selector to address.